colcon/install.yaml

```diff
@@ -0,0 +1,2 @@
+- type: system
```

Comment:
I think this requires either ros-setup or ros2-setup

Reply:
Done